package.json

```diff
@@ -11,6 +11,8 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@chakra-ui/icon": "^3.0.16",
```

Comment:
We can remove this if it isn't being used

Reply:
it's being used in `SearchBar.tsx`